dockerfiles/interop/src/env.yaml

```diff
@@ -0,0 +1,26 @@
+application:
+  hostname: REPLACE_HOSTNAME
+  ocphostname: REPLACE_OCP_HOSTNAME
+  ocpsecurehostname: REPLACE_OCP_SECURE_HOSTNAME
+  password: password
```

Comment:
We discussed hiding plaintext passwords is there a way we can do that here like we've done with the FTP server creds below? Is there a reason we may not want to?

Reply:
@ascerra I think these are defaults for the operator, but they may not even be used anymore. @sshveta may be able to confirm. I will add them to vault for now.

Reply:
They are used @calebevans and yes they are default